tools/gen_runtime_vectors/main.go

```diff
@@ -193,6 +193,80 @@ func main() {
 						}
 						vectors = append(vectors, MakeRuntimeTestVector(tx, txBody, meta, t.valid, t.signer, nonce))
 					}
+
+					// consensusaccounts.Delegate
+					for _, t := range []struct {
+						to           string
+						origTo       *ethCommon.Address
+						chainContext string
+						valid        bool
+					}{
+						// Valid Delegate: Alice -> Bob's native address on ParaTime
+						{testing.Bob.Address.String(), nil, context.ChainContext, true},
+						// Valid Delegate: Alice -> Dave's native address on ParaTime
+						{testing.Dave.Address.String(), nil, context.ChainContext, true},
+						// Valid Delegate: Alice -> Dave's ethereum address on ParaTime
+						{testing.Dave.EthAddress.String(), testing.Dave.EthAddress, context.ChainContext, true},
```

Comment:
Just note that delegating to Ethereum addresses makes absolutely no sense as those are not controllable in the consenus layer.

Reply:
Is there a reason to keep this at all or should we just remove non-consensus addresses? If so, I would also remove orig_to from meta for delegate/undelegate?

Reply:
I removed the non-consensus addresses now. Still not sure, if we may keep the support for signing those or they open potential vulnerabilities by supporting it.